### DISCUSSION TITLE
Is it possible to make strimizi operator version supports kafka-2.7.2？

### DISCUSSION THREAD

**Discussion by yaoyao12138** [strimzi/strimzi-kafka-operator#6726] [Q&A]
We need strimizi kafka operator support kafka-2.7.2, and I checked the discussion  https://github.com/strimzi/strimzi-kafka-operator/discussions/6640, unfortunately it doesn't support.   

Is it possible to make strimizi operator version supports kafka-2.7.2 by contribution through us?    
Besides added kafka version like this here https://github.com/strimzi/strimzi-kafka-operator/blob/main/kafka-versions.yaml#L123-L131,  and other two places https://github.com/strimzi/strimzi-kafka-operator/blob/c5af742634293465945e07bc9bbd62053e343151/packaging/helm-charts/helm3/strimzi-kafka-operator/templates/_kafka_image_map.tpl#L18 
https://github.com/strimzi/strimzi-kafka-operator/blob/c5af742634293465945e07bc9bbd62053e343151/packaging/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml#L62
what else should we do? Do we need follow this PR? https://github.com/strimzi/strimzi-kafka-operator/pull/6482/files   
Thank you very much! @scholzj

**Comment by scholzj** [comment]
Kafka 2.7 is quite old and not supported by latest Strimzi versions. You would need to add it your self to one of the old versions and build it your self. The PR shows what would need to be changed.
